### PR TITLE
[FSDE-279] Add interactive documentation examples of Chi Vue data table

### PR DESCRIPTION
**Pull request by janibolkvadze**

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f6091d06c830f59a1c547a57e9f71a624fd34991-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://25288b1b435dcf3352512f73a6ca644ed32ec132-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://2cca868fe060f4c3278881131cd1bc80328c861a-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f83f8c17f75ca7acabac462f893ace28a4911045-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/5/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/6/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://c2a506d193afa297609a6916db1eda92809d2c9c-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f097370cb5d7072a05635e83b583e9397f1c92cc-chi-pr.enterprise.ctl.io

File: src/website/views/components/data-table/examples/base/_sorting-ascending.pug [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+            portal: false,
+            noBorder: false,
+            bordered: false,
+            hover: false,
+            size: 'md',
+            striped: false,
+          },
+            defaultSort: {
+            key: 'name',
+            direction: 'ascending',
```
Comment (jllr) [inline src/website/views/components/data-table/examples/base/_sorting-ascending.pug:37]: @janibolkvadze please indent these 2 lines.

File: src/website/views/components/data-table/examples/portal/_complex.pug [hunk 2]
```diff
@@ -0,0 +1,1708 @@
...
+                </div>
+              </div>
+            </template>
+
+            <!-- mounted -->
+            mounted() {
+              chi.dropdown(this.$refs.dropdownTrigger);
+            }
+  .chi-tabs-panel#html-portal-complex
+    :code(lang="html")
```
Comment (jllr) [inline src/website/views/components/data-table/examples/portal/_complex.pug:843]: @janibolkvadze I would suggest removing html blueprint as it doesn't match with the Vue code example

**Review by jllr — CHANGES_REQUESTED** [review]
@janibolkvadze Portal examples row height should be 56px as per brand guidelines. Could you change it?

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/8/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/9/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://d1ed7a144303d5c348fce6c1009b0f7223b5b3cf-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/10/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://a309525273de414d9d1ef3bbcd79100c7bd11f0d-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/11/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://7263244d348ff90086f7e3313372b79914d8898d-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/12/. :x:

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/13/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-79/14/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://256af5c76caa6a58c55d2b647a25d1a24da9668d-chi-pr.enterprise.ctl.io

**Review by aryaroudi — APPROVED** [review]

**Pull request merged**